lib/routers/establishment/invite-user.js

```diff
@@ -23,14 +22,15 @@ const create = (req, res, next) => {
 const validateInvitation = (req, res, next) => {
   return validateSchema(req.models.Invitation, {
     token: 'abc123',
-    email: req.body.email,
+    email: req.body.data.email,
     establishmentId: req.establishment.id,
-    role: req.body.role
+    role: req.body.data.role
   })(req, res, next);
 };
 
 router.post('/',
   permissions('profile.invite'),
+  whitelist('email', 'role', 'firstName', 'lastName'),
```

Comment:
we possibly want a config file to define whitelisted properties

Reply:
I'm not certain I agree right now tbh. They're not shared across files at all, and only used once or twice per file. It feels like it would just spread things around for the sake of it.

Reply:
FINE.